### PR TITLE
Build tests with upstream Quincy build and repos.

### PR DESCRIPTION
**Pull request by sunilkumarn417**
Signed-off-by: sunilkumarn417 <sunnagar@redhat.com>

- Added config and test suite directories and also initial tier-0 test suite.
- Code changes to accept upstream build and execute deployment test cases.

**test results:**
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-GWVOZB

### CODE REVIEW

File: conf/quincy/cephadm/sanity-cephadm.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+globals:
```
Comment (psathyan) [inline conf/quincy/cephadm/sanity-cephadm.yaml:1]: Could you add some documentation about the cluster layout?

IMO, could we avoid adding folder structure here and opt for using an appropriate cluster naming. For example
```
9-node-all-roles.yaml
```

File: suites/quincy/cephadm/tier-0_cephadm.yaml [hunk 2]
```diff
@@ -0,0 +1,814 @@
+#===============================================================================================
```
Comment (psathyan) [inline suites/quincy/cephadm/tier-0_cephadm.yaml:1]: Could we avoid using `tier-0`? This does not match with the Polarion tier levels. 

Instead opt to name the file that is relevant to what is being tested. Example
`tier-1_dfg-test-cephadm-regression`.

We could also split them into multiple suites that focus on a specific command or workflow

File: conf/quincy/cephadm/sanity-cephadm.yaml (hunk 1 above)
Comment (sunilkumarn417) [reply conf/quincy/cephadm/sanity-cephadm.yaml]: done

**Comment by sunilkumarn417** [comment]
Test results:
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-WUTG7I/  ( 1 test failed )
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-DQSWT0 (Fixed failed test case)

**Review by psathyan — APPROVED** [review]

**Pull request merged**